src/constants/Style.js

```diff
@@ -140,6 +140,8 @@ module.exports = {
       windowSize = 'large';
     } else if (width >= breakPoints.medium) {
       windowSize = 'medium';
+    } else if (width <= breakPoints.small) {
+      windowSize = 'xsmall';
```

Comment:
I like it 👍 

I just added `getWindowSize` to the docs (#411). Please add something about the new `xsmall` size when you add your changes to the change log.